packages/ui/src/Dashboard/DetailView/ActionsDetails/BuySell/BuySell.tsx

```diff
@@ -52,27 +56,23 @@ class BuySell extends Component<Props, State> {
       const response = await apiCall('/v1/order/place', 'POST', payload);
       const error = getErrorMessage(response);
       if (error) {
-        console.log('Error');
-        console.log(error);
-        console.log(payload);
-        console.log(response);
-      } else {
-        console.log('Placed');
-        console.log(payload);
-        console.log(response);
+        this.props.showMessagePopup('error', response.errors.join(' , '));
+      }
+      else {
+        this.props.showMessagePopup('success', response.data.status);
+        this.props.fetchUserDetails();
```

Comment:
is 'await' required here, since it's an async function?
Can you check if the user state is getting updated with the new order?

Reply:
Checked👍. State is getting updated.